packages/design-system/src/common/Spacing.ts

```diff
@@ -38,3 +38,5 @@ export type Spacing =
   // for margins only
   | 'form-panel-subsection'
   | 'form-action-buttons'
+  | string
+  | number
```

Comment:
Adding this because we are using `strings`, 'boolean` and number as Spacing values in NextGenUi. Type check was failing because of this after adding @harness/design-system as a package

Reply:
@abhinavrastogi-harness @vkbansal